string joint_name
float64 position
float32 speed
float32 acceleration
